neqo-transport/src/cc/new_reno.rs

```diff
@@ -0,0 +1,44 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Congestion control
+#![deny(clippy::pedantic)]
+
+use std::fmt::{self, Display};
+
+use crate::cc::{CwndFn, MAX_DATAGRAM_SIZE};
+use neqo_common::qinfo;
+
+#[derive(Debug)]
+pub struct NewReno {}
+
+impl Default for NewReno {
+    fn default() -> Self {
+        Self {}
+    }
+}
+
+impl Display for NewReno {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "NewReno",)?;
```

Comment:
autoformat for macros is poor:

```suggestion
        write!(f, "NewReno")?;
```